roles/wordpress/tasks/manage_plugins.yml

```diff
@@ -0,0 +1,73 @@
+---
+
+- name: Get list of installed WordPress plugins
+  command: 'wp plugin list --format=json'
+  sudo_user: '{{ wordpress_user }}'
+  args:
+    chdir: '{{ wordpress_root }}'
+  changed_when: False
+  always_run: True
+  register: wordpress_plugin_list_installed_cmd
+
+  ## FIXME(ypid): Revert this file back to c4e819a020f1c2e7094f5b8f5aaa180678163642 once
+  ## selectattr("status", "equalto", "active") can be used.
+- name: Get list of active WordPress plugins
+  command: 'wp plugin list --format=json --status=active'
+  sudo_user: '{{ wordpress_user }}'
+  args:
+    chdir: '{{ wordpress_root }}'
+  changed_when: False
+  always_run: True
+  register: wordpress_plugin_list_active_cmd
+
+- name: Convert list of installed WordPress plugins from JSON
```

Comment:
Any reason why this is its own step? I tried combining the two like `wordpress_plugin_list_installed_cmd.stdout|from_json|map(attribute="name")|list`. It seemed to work fine.